### DISCUSSION TITLE
Why there are two repos, autogen_agentchat and autogen?

### DISCUSSION THREAD

**Discussion by MkYCHsu** [microsoft/autogen#3914] [General]
Could anyone please explain to me that why they are developing these two repos simultaneously?

https://github.com/microsoft/autogen
and
https://github.com/autogenhub/autogen

Also, most of the links you can find on google for the source code of autogen just doesn't work... It's everywhere. 
For example,
https://github.com/microsoft/autogen/blob/main/autogen/agentchat/assistant_agent.py

I just don't understand why making things this complicated.

**Comment by jackgerrits** [comment]
Hi @MicDonald,

You can find explanation [here](https://github.com/microsoft/autogen/discussions/3697#discussioncomment-10895954) and in the [FAQs in the readme](https://github.com/microsoft/autogen?tab=readme-ov-file#faqs). In particular [this answer](https://github.com/microsoft/autogen?tab=readme-ov-file#what-about-forks).

For that link, we've moved development of 0.2 to the 0.2 branch while 0.4 is worked on in the main branch. If you switch the branch selector for that file over to 0.2 you can see it. (https://github.com/microsoft/autogen/blob/0.2/autogen/agentchat/assistant_agent.py)
